.gitignore

```diff
@@ -2,3 +2,7 @@ composer.lock
 phpunit.xml
 vendor
 
```

Comment:
The .gitignore shouldn't be modified in any way

Reply:
Argh! wasn't meant to be included in the commit! - Bloody PHPStorm!